uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/prebookconfirmation/PrebookConfirmationView.kt

```diff
@@ -1,102 +1,112 @@
 package com.karhoo.uisdk.screen.booking.checkout.prebookconfirmation
 
-import android.content.Context
-import android.util.AttributeSet
+import android.os.Bundle
+import android.view.LayoutInflater
 import android.view.View
-import android.widget.FrameLayout
-import androidx.core.content.ContextCompat.startActivity
-import com.karhoo.sdk.api.model.PickupType
+import android.view.ViewGroup
+import android.widget.ImageButton
+import android.widget.ImageView
+import android.widget.TextView
+import com.karhoo.sdk.api.model.Quote
 import com.karhoo.sdk.api.model.QuoteType
 import com.karhoo.sdk.api.model.TripInfo
 import com.karhoo.uisdk.R
 import com.karhoo.uisdk.base.ScheduledDateView
 import com.karhoo.uisdk.base.ScheduledDateViewBinder
-import com.karhoo.uisdk.base.dialog.KarhooAlertDialogAction
-import com.karhoo.uisdk.base.dialog.KarhooAlertDialogConfig
-import com.karhoo.uisdk.base.dialog.KarhooAlertDialogHelper
+import com.karhoo.uisdk.base.bottomSheet.MasterBottomSheetFragment
 import com.karhoo.uisdk.screen.booking.checkout.component.views.CheckoutViewContract
-import com.karhoo.uisdk.screen.rides.detail.RideDetailActivity
+import com.karhoo.uisdk.screen.booking.domain.quotes.VehicleMappingsProvider
 import com.karhoo.uisdk.util.DateUtil
+import com.karhoo.uisdk.util.PicassoLoader
+import com.karhoo.uisdk.util.extension.getCorrespondingLogoMapping
 import com.karhoo.uisdk.util.extension.orZero
 import com.karhoo.uisdk.util.extension.toLocalisedString
 import com.karhoo.uisdk.util.formatted
-import kotlinx.android.synthetic.main.uisdk_alert_prebook_confirmation.view.bookingDateText
-import kotlinx.android.synthetic.main.uisdk_alert_prebook_confirmation.view.bookingTimeText
-import kotlinx.android.synthetic.main.uisdk_alert_prebook_confirmation.view.dropoffAddressText
-import kotlinx.android.synthetic.main.uisdk_alert_prebook_confirmation.view.fareText
-import kotlinx.android.synthetic.main.uisdk_alert_prebook_confirmation.view.fareTypeText
-import kotlinx.android.synthetic.main.uisdk_alert_prebook_confirmation.view.meetingPointText
-import kotlinx.android.synthetic.main.uisdk_alert_prebook_confirmation.view.pickupAddressText
-import kotlinx.android.synthetic.main.uisdk_alert_prebook_confirmation.view.pickupTypeText
 import org.joda.time.DateTime
-import java.util.Currency
+import java.util.*
 
-class PrebookConfirmationView @JvmOverloads constructor(
-    context: Context,
-    attrs: AttributeSet? = null,
-    defStyleAttr: Int = 0)
-    : FrameLayout(context, attrs, defStyleAttr), ScheduledDateView {
+class PrebookConfirmationView(val quoteType: QuoteType?, val trip: TripInfo, val quote: Quote?) :
```

Comment:
The component is now named `BookingConfirmation`. We do show it for prebook only at this moment, so the behaviour is the same as before.

Reply:
Shall we rename it ?